src/app/base/components/record-display/record-display.component.html

```diff
@@ -13,6 +13,7 @@
         </ng-template>
         <ng-template #noHighlight>
             <span
+                [ngClass]="recordDisplayOptions.dataBrowserModals.configuration.lineBreaks.normal ? recordDisplayOptions.dataBrowserModals.configuration.lineBreaks.isPreWrap? 'whitespace-pre-wrap':'whitespace-pre-line' :''"
```

Comment:
condition looks a bit weird to me. I read it as if its normal we then decide between pre wrap & pre line (since ? is <condition> ? <true> : <false>. maybe some brackets or !normal